### PR TITLE
bye bye symfony wheels

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A                                                |
|---          |---                                              |
|Fixed Tickets|fixed the coupling in symfony console application|
|License      |MIT                                              |
                                                                 

uses webmozart to stay without symfony wheels

### CODE REVIEW

**Comment by sstok** [comment]
You known internally it still uses Symfony for some things ;) but we don't bother with that.

File: .travis.yml [hunk 1]
```diff
@@ -10,19 +10,7 @@
 language: php
 
 php:
-    - 5.4
```
Comment (GrahamCampbell) [inline .travis.yml]: What's going on here>

Comment (cordoval) [inline .travis.yml]: is just to focus :+1: speed up travis and count what matters the most for this branch

Comment (cordoval) [inline .travis.yml]: @GrahamCampbell would be good to hear you are using gush :+1: :older_man: would be rewarding, i know you work with laravel but still, would be nice to hear if some people in the l5 community are using it :open_mouth:

Comment (GrahamCampbell) [inline .travis.yml]: We're not using it atm, though I would like to in the future.

Comment (cordoval) [inline .travis.yml]: please do :+1:

**Comment by cordoval** [comment]
sorry did not have enough time to complete this, closing for the moment

**Pull request closed without merging**